types/fabric/fabric-impl.d.ts

```diff
@@ -293,9 +293,9 @@ interface ICollection<T> {
 
     /**
      * Returns object at specified index
-     * @return thisArg
+     * @return object at index
      */
-    item(index: number): T;
+    item(index: number): Object;
```

Comment:
I think this defeats the point of the generic. Maybe we should just enforce that the generic type must be extend Object?